src/server/slimredis/data/cmd_bitmap.c

```diff
@@ -75,7 +76,9 @@ _add_key(struct response *rsp, struct bstring *key)
 
         return NULL;
     } else { /* cuckoo insert current won't fail as long as size is valid */
-        it = cuckoo_insert(key, NULL, time_reltime(0));
+        /* Currently, expiry is set as a big number as a workaround to mean
+           never expire */
+        it = cuckoo_insert(key, NULL, INT32_MAX - 1);
```

Comment:
Not your change- but I wonder if it's better to just set it to INT32_MAX, but maybe it's not that important...

Reply:
filed #189 to look into this.